GSA/made-in-america
# [MIA-441] Financial Assistance page updates

## Description

The client asked for the page to be updated... the table was made into one column and hyperlinks are the names of the agencies along with an update to the top text

## How to test

- pull down branch
- run `npm run start`
- vist `http://localhost:4000/financial-assistance/` and check that table has been adjusted 

To prepare for code review, please follow the following checks.
